examples/containerlab/README.md

```diff
@@ -0,0 +1,60 @@
+# Containerlab example
+
+This folder contains a simple containerlab topology which is using the image created in the main README.
+The purpose of this lab is to test the creation of bond interface with the API.
+
+## Deployement
+
+Deploy the topology:
+
+```
+sudo clab deploy -t topology.clab.yml
+```
+
+## Manual setup
+
+The `setup_hosts` script configure the bonding automatically, and assign an ip address for the `bond0` interface on each side.
```

Comment:
Why doing this with a script? The whole point of the api is that everything can be done using the api instead of cli.  Isn't it how you tried it?  This example should showcase the api, not how the ip command works (every one already know this one)

Reply:
This script was just there to show what the API was actually doing but I will remove it.